fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSites.kt

```diff
@@ -227,17 +229,47 @@ private fun TopSiteFaviconCard(topSite: TopSite) {
         Box(contentAlignment = Alignment.Center) {
             Surface(
                 modifier = Modifier.size(36.dp),
+                color = FirefoxTheme.colors.layer2,
                 shape = RoundedCornerShape(4.dp),
             ) {
-                Favicon(
-                    url = topSite.url,
-                    size = 36.dp,
-                )
+                when (topSite.url) {
+                    SupportUtils.POCKET_TRENDING_URL,
+                    SupportUtils.BAIDU_URL,
+                    SupportUtils.JD_URL,
+                    SupportUtils.PDD_URL,
+                    SupportUtils.TC_URL,
+                    SupportUtils.MEITUAN_URL,
+                    -> {
+                        Image(
+                            painter = painterResource(id = getDrawableForUrl(topSite.url)!!),
+                            contentDescription = null,
+                            modifier = Modifier.size(36.dp),
```

Comment:
💭 think we should extract the common dimensions as constants that being used in multiple places - 36, 84 etc.

Reply:
@rahulsainani Are you referring to this specific component (eg, whole file) or a more broad dimens definition?

Reply:
Coffee hasn't kicked in yet, but I think you're in fact referring to this file.

Reply:
Yeah, for this file only, since we see `36.dp` and `84.dp` a few times.

Reply:
Added constants for common sizes.